<ui version="4.0" >
 <class>SwarmVisClass</class>
 <widget class="QMainWindow" name="SwarmVisClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1068</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title" >
     <string>OpenGL Window</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>410</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title" >
     <string>Playback</string>
    </property>
    <widget class="QPushButton" name="btnStop" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>30</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>Play</string>
     </property>
    </widget>
    <widget class="QSlider" name="timeSlider" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSpinBoxDelay" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>140</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum" >
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Frame Delay in ms:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab" >
     <attribute name="title" >
      <string>Display</string>
     </attribute>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Point Size:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinBoxPointSize" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>40</y>
        <width>62</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum" >
       <double>0.010000000000000</double>
      </property>
      <property name="singleStep" >
       <double>0.100000000000000</double>
      </property>
      <property name="value" >
       <double>3.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Agent Trail Length:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxTrailLength" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2" >
     <attribute name="title" >
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>600</y>
      <width>801</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title" >
     <string>Display Options</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1068</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit" />
    <addaction name="action_Load_Agent_Data" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Load_Agent_Data" >
   <property name="text" >
    <string>&amp;Load Agent Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked()</signal>
   <receiver>SwarmVisClass</receiver>
   <slot>btnPlay_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>130</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>49</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>SwarmVisClass</receiver>
   <slot>menubar_action_handler(QAction*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>15</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel" >
     <x>239</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStop</sender>
   <signal>clicked()</signal>
   <receiver>SwarmVisClass</receiver>
   <slot>btnStop_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>72</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnPlay_clicked()</slot>
  <slot>menubar_action_handler(QAction*)</slot>
  <slot>btnStop_clicked()</slot>
  <signal>playSignal()</signal>
 </slots>
</ui>
